ebpfapi/platform.cpp

```diff
@@ -54,4 +55,25 @@ CloseHandle(_In_ _Post_ptr_invalid_ ebpf_handle_t handle)
 {
     return ::CloseHandle(handle);
 }
+
+int
+_open_osfhandle(intptr_t os_file_handle, int flags)
```

Comment:
since ebpf_handle.h has
```
typedef uint64_t ebpf_handle_t;
```
i.e.,unsigned, I'd suggest uintptr_t instead of intptr_t throughout, for consistency
```suggestion
_open_osfhandle(uintptr_t os_file_handle, int flags)
```

Reply:
The official definition of [_open_osfhandle](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/open-osfhandle?view=msvc-160)  has intptr_t, hence making this intptr_t as well.

Reply:
ah.  should we update ebpf_handle.h then to be intptr_t?